$NetBSD: patch-pcbnew_CMakeLists.txt,v 1.3 2016/09/08 09:32:28 bouyer Exp $

Don't use ${CMAKE_MODULE_PATH} directly as it will be changed
and containt multiple directories

Force proper LDFLAGS for MODULE, as cmake doens't set rpath for theses

--- pcbnew/CMakeLists.txt.orig	2015-12-09 17:21:54.000000000 +0100
+++ pcbnew/CMakeLists.txt	2015-12-16 14:52:03.000000000 +0100
@@ -521,7 +521,7 @@
     COMMAND ${CMAKE_COMMAND}
         -DinputFile=${CMAKE_CURRENT_SOURCE_DIR}/dialogs/dialog_freeroute_exchange_help.html
         -DoutputFile=${CMAKE_CURRENT_SOURCE_DIR}/dialogs/dialog_freeroute_exchange_help_html.h
-        -P ${CMAKE_MODULE_PATH}/Html2C.cmake
+        -P ${PROJECT_SOURCE_DIR}/CMakeModules/Html2C.cmake
     DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/dialogs/dialog_freeroute_exchange_help.html
     COMMENT "creating ${CMAKE_CURRENT_SOURCE_DIR}/dialogs/dialog_freeroute_exchange_help_html.h
        from ${CMAKE_CURRENT_SOURCE_DIR}/dialogs/dialog_freeroute_exchange_help.html"
@@ -613,6 +613,9 @@
         LINK_FLAGS "${TO_LINKER},-cref ${TO_LINKER},-Map=pcbnew.map"
         )
 endif()
+
+set_property(TARGET pcbnew_kiface APPEND_STRING
+	PROPERTY LINK_FLAGS " $ENV{LDFLAGS}")
 
 # if building pcbnew, then also build pcbnew_kiface if out of date.
 add_dependencies( pcbnew pcbnew_kiface )
